SPECS/tdnf/tdnf.spec

```diff
@@ -167,6 +166,19 @@ if [[ ! -f %{_tdnf_history_db_dir}/history.db ]]; then
     %{_libdir}/tdnf/tdnf-history-util init
 fi
 
+# Make the decision to update yum sym link during post script
+# not package build. Some workloads want to disable tdnf, so
+# decision should not be made during build but during install
+# of package.
+if [ ! -h $(rpm --eval %{_bindir})/yum ]; then
```

Comment:
There are 2 problems on the upgrade operation from previous behavior to this new behavior.
1. Since the older tdnf with the `yum` symlink is still around at time of installation of new `tdnf` and this `%post` script runs, the conditional fails and the body of this script never runs so the `yum` symlink is never created. This may be acceptable behavior.
2. Then later in the transaction, the old `tdnf` package gets deleted, which takes away the `tdnf` symlink. Therefore at the end of the transaction, there is no symlink at all

Thinking about moving this logic to `%posttrans` which runs at the very end of the transaction to ensure we always have a symlink.